### ISSUE TITLE
Manually input data?

### ISSUE THREAD

**Issue by sblanky** [AIF-development-team/adsorptioninformationformat#26]
Wondering if it would be possible to add a feature whereby you could simply copy and paste the adsorption data (say from CSV) on the GUI? If data is not in one of the supported formats, the parser doesn't work. 

Obviously this can be done by manipulating an aif file itself, but may improve workflows if it could be done in the GUI.

**Comment by jackevansadl** [comment]
Absolutely, this is one of the paths forward we are currently investigating. One of the problems will be the units aren't necessarily included in a simple csv file. As a result we are adapting some GUI submission pages to enable user inputed csv along with metadata fields such as units, temperature etc.

In the meantime, I would suggest using a previously reported format (xls or csv) replacing the pressure and amount with the data you wish to use. This way you can use this file as a package to also pass important metadata to the AIF output.
